bundles/org.openhab.ui/web/src/components/developer/developer-sidebar.vue

```diff
@@ -659,22 +704,15 @@ export default {
       if (this.searchResultsLoading) return
 
       const promises = (this.cachedObjects)
-        ? [
-          Promise.resolve(this.cachedObjects[0]),
-          Promise.resolve(this.cachedObjects[1]),
-          Promise.resolve(this.cachedObjects[2]),
-          Promise.resolve(this.cachedObjects[3]),
-          Promise.resolve(this.cachedObjects[4]),
-          Promise.resolve(this.cachedObjects[5]),
-          Promise.resolve(this.cachedObjects[6])
-        ] : [
-          this.$oh.api.get('/rest/items?staticDataOnly=true&metadata=.*'),
-          this.$oh.api.get('/rest/things?summary=true'),
-          this.$oh.api.get('/rest/rules?summary=false'),
-          Promise.resolve(this.$store.getters.pages),
-          this.$oh.api.get('/rest/transformations'),
-          this.$oh.api.get('/rest/ui/components/system:sitemap'),
-          this.loadPersistenceConfigs()
+        ? this.cachedObjects.map((o) => Promise.resolve(o)) : [
+          this.$oh.api.get('/rest/items?staticDataOnly=true&metadata=.*'), // 0
+          this.$oh.api.get('/rest/things?summary=true'), // 1
+          this.$oh.api.get('/rest/rules?summary=false'), // 2
+          Promise.resolve(this.$store.getters.pages), // 3
+          this.$oh.api.get('/rest/ui/components/system:sitemap'), // 4
+          this.$oh.api.get('/rest/ui/components/ui:widget'), // 5
```

Comment:
It is possible to get the widgets from Vuex:

```suggestion
          Promise.resolve(this.$store.getters.widgets), // 5
```